src/features/donation/components/modal-content.tsx

```
@@ -86,7 +86,7 @@ export const DonationModalContent: React.FC<DonationModalContentProps> = ({
   const { isLoading: isDonationConfigLoading, data: donationConfig } =
     donationContractHooks.useConfig();
 
-  const { form, matchingPots, isDisabled, onSubmit, totalAmountFloat, isGroupDonation } =
+  const { form, matchingPots, isDisabled, onSubmit, totalAmountFloat, isGroupDonation, crossChainMinAmount, crossChainTokenSymbol } =
     useDonationForm(props);
```

ESLint

[error] Replace `·form,·matchingPots,·isDisabled,·onSubmit,·totalAmountFloat,·isGroupDonation,·crossChainMinAmount,·crossChainTokenSymbol·}·=⏎···` with `⏎····form,⏎····matchingPots,⏎····isDisabled,⏎····onSubmit,⏎····totalAmountFloat,⏎····isGroupDonation,⏎····crossChainMinAmount,⏎····crossChainTokenSymbol,⏎··}·=`

(prettier/prettier)